ai.py

```diff
@@ -80,7 +80,7 @@ def select_card(enemy_card):
     if enemy_card != "player":
         # If opponent has more of not void  cardboxes
         for card in self_cards:
-            if card.type is 'warrior_card':
+            if card.type ==a 'warrior_card':
```

Comment:
Typo is here

Reply:
I was pretty sure I fixed it but looks like I missed to push, sorry for that.